packages/platform_maps_flutter_apple/lib/src/apple_maps_widget.dart

```diff
@@ -0,0 +1,217 @@
+  bool get trafficEnabled => params.trafficEnabled;
+
+  /// Which gestures should be consumed by the map.
+  ///
+  /// It is possible for other gesture recognizers to be competing with the map on pointer
+  /// events, e.g if the map is inside a [ListView] the [ListView] will want to handle
+  /// vertical drags. The map will claim gestures that are recognized by any of the
+  /// recognizers on this list.
+  ///
+  /// When this set is empty, the map will only handle pointer events for gestures that
+  /// were not claimed by any other gesture recognizer.
+  Set<Factory<OneSequenceGestureRecognizer>> get gestureRecognizers =>
+      params.gestureRecognizers;
+  @override
+  _PlatformMapState createState() => _PlatformMapState();
+}
+
+class _PlatformMapState extends State<_PlatformMap> {
+  @override
+  Widget build(BuildContext context) {
+    return apple_maps.AppleMap(
+      initialCameraPosition:
+          widget.initialCameraPosition.appleMapsCameraPosition,
+      compassEnabled: widget.compassEnabled,
+      mapType: _getMapType(),
+      padding: widget.padding,
+      annotations: widget.markers.appleMapsAnnotationSet,
+      polylines: widget.polylines.appleMapsPolylineSet,
+      polygons: widget.polygons.appleMapsPolygonSet,
+      circles: widget.circles.appleMapsCircleSet,
+      gestureRecognizers: widget.gestureRecognizers,
+      onCameraIdle: widget.onCameraIdle,
+      myLocationButtonEnabled: widget.myLocationButtonEnabled,
+      myLocationEnabled: widget.myLocationEnabled,
+      onCameraMoveStarted: widget.onCameraMoveStarted,
+      pitchGesturesEnabled: widget.tiltGesturesEnabled,
+      rotateGesturesEnabled: widget.rotateGesturesEnabled,
+      zoomGesturesEnabled: widget.zoomGesturesEnabled,
+      scrollGesturesEnabled: widget.scrollGesturesEnabled,
+      onMapCreated: _onMapCreated,
+      onCameraMove: _onCameraMove,
+      onTap: _onTap,
+      onLongPress: _onLongPress,
+      trafficEnabled: widget.trafficEnabled,
+      minMaxZoomPreference: widget.minMaxZoomPreference.appleMapsZoomPreference,
+    );
+  }
+
+  void _onMapCreated(apple_maps.AppleMapController controller) {
+    widget.onMapCreated?.call(controller.platformMapController);
+  }
+
+  void _onCameraMove(apple_maps.CameraPosition cameraPosition) {
+    widget.onCameraMove?.call(cameraPosition.platformCameraPosition);
+  }
+
+  void _onTap(apple_maps.LatLng position) {
+    widget.onTap?.call(position.platformLatLng);
+  }
+
+  void _onLongPress(apple_maps.LatLng position) {
+    widget.onLongPress?.call(position.platformLatLng);
+  }
+
+  apple_maps.MapType _getMapType() {
+    if (widget.mapType == MapType.normal) {
+      return apple_maps.MapType.standard;
+    } else if (widget.mapType == MapType.satellite) {
+      return apple_maps.MapType.satellite;
+    } else if (widget.mapType == MapType.hybrid) {
+      return apple_maps.MapType.hybrid;
+    }
+    return apple_maps.MapType.standard;
```

Comment:
```suggestion
    switch (widget.mapType) {
        case MapType.normal:
          return apple_maps.MapType.standard;
        case MapType.satellite:
          return apple_maps.MapType.satellite;
        case MapType.hybrid:
          return apple_maps.MapType.hybrid;
  }
```
I do not know if you want to apply some code changes already. In case here is one.